rlclientlib/logger/logger_extensions.cc

```diff
@@ -38,10 +40,9 @@ i_logger_extensions::~i_logger_extensions() { }
 
 
 i_logger_extensions* i_logger_extensions::get_extensions(const utility::configuration& config, i_time_provider* time_provider) {
-	const bool enable_dedup = config.get_int(name::PROTOCOL_VERSION, 1) == 2 &&
-		to_content_encoding_enum(config.get(name::INTERACTION_CONTENT_ENCODING, value::CONTENT_ENCODING_IDENTITY)) == content_encoding_enum::ZSTD_AND_DEDUP;
-
-	return enable_dedup ? create_dedup_logger_extension(config, time_provider) : new default_extensions(config, time_provider);
+	const char *section = "interaction"; //fixme lift this to live_model_impl;
```

Comment:
Is this fixme still valid?

Reply:
Yes, it is. We should lift that to live_model_impl and make it so that extensions can be applied to observations as well.
I left as a FIXME because neither of the current extensions matter for observations since the payloads are too small to compress and are not dedup'able.